Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -76,6 +79,10 @@
     <Word from="deo" to="dio" />
     <Word from="Deo" to="Dio" />
     <Word from="dešava" to="događa" />
+    <Word from="dete" to="dijete" />
+    <Word from="djete" to="dijete" />
+    <Word from="Dete" to="Dijete" />
+    <Word from="Djete" to="Dijete" />
```

Comment:
I know you meant well, @ivandrofly , but if you look closely, these words are all in a **wordlist**, not in regex, so it's not really possible to merge them like that.